src/components/custom-field-input-date/custom-field-date.css

```diff
@@ -0,0 +1,21 @@
+@import '../../styles/spacing.css';
+@import '../../styles/colors-v2.css';
+
+.input-icon {
+  composes: input-icon from '../__internal__/abstract-custom-field/abstract-custom-field.css'; /* stylelint-disable-line css-modules/composed-class-names */
+  pointer-events: auto;
+  height: 18px;
+  width: 18px;
+}
+
+.date-input input::-webkit-inner-spin-button,
```

Comment:
Does this work in Chrome, Firefox, Safari, and/or Edge?

Reply:
I'm gonna guess it doesn't. Do we not have an autoprefixer for these "experimental" selectors?

Reply:
It works on chrome, firefox, safari (I haven't yet tested Edge)

Reply:
It works in Edge too 🥳